packages/ui-extensions/src/types/Heading.ts

```diff
@@ -0,0 +1,20 @@
+type Level = 1 | 2 | 3;
+
+export interface BaseHeadingProps {
+  /**
+   * Unique identifier. Typically used to make the heading a target
+   * that another component can refer to in order to provide an alternative
+   * accessibility label.
+   */
+  id?: string;
```

Comment:
This field will potentially be used by a partner?

Reply:
Yes, a partner may want to use `aria-labelledby` and refer to this id.